### DISCUSSION TITLE
nice ideas for the future update!?!

### DISCUSSION THREAD

**Discussion by Shzv** [diamond-rip/Eden#95] [Ideas]
Hope to see the features mentioned in the future
@GoodestEnglish 

• Added /leave command to quit the game
• Added the /queue command for the kit, example: /queue Boxing
• add the ability for VIPs or staff to see players in the lobby with permission
• the possibility to disable the drop (type q or ctrl + q with the keyboard) in the game rules
• add kill (placeholder as mmc) in bridge mode and bedwars if needed (tried with placeholder but it works)
• addition of a combo counter in boxing mode like on minemen (reason: it's cool)
• added ping range in classified scoreboard
• add the ability to put a custom command in spawn items

• changing portal security
  such as the minimum and maximum setting.
  since in many maps it would be useful and would help to have a better setting (modification or addition, the addition would be 
  better)

I hope to see these modes in the future. the ones that take more time in my opinion 

• adding FireBall fight mode in the future
• adding placeholders to make other plugins compatible example: %queue_player% (it would help to create menus with other plugins and create leaderboard holograms)
• added Winstreak and Daily Streak for modes

**Comment by HierarchyMan** [comment]
+1
